packages/components-props-analyzer/src/ComponentsList.ts

```diff
@@ -0,0 +1,40 @@
+export interface Component {
+    /**
+     * The name of the component as it is exported
+     */
+    name: string;
+    /**
+     * the name of the package from which the component is exported
+     */
+    packageName: string;
+}
+
+export interface ComponentMetadata {
+    name: string;
+    description: string;
+    type: string;
+    optional: boolean;
+    defaultValue: string | null;
+    deprecation: string | null;
+    params: Array<{parameterName: string; text: string}>;
+}
+
+/**
+ * These are the components whose props are going to be automatically extracted into ./src/components
```

Comment:
Why those 3 only? 🤔

Reply:
I used those to test out my package and my PR was already getting pretty big. Next PR should contain all of them